terraform/bastion/Jenkinsfile

```diff
@@ -43,7 +52,7 @@ pipeline {
     stage('Create user if creating bastion') {
```

Comment:
Maybe change the `stage` description as this only applies if connecting to the DB, not the other connections: "Create user if connecting to database"

Reply:
Makes sense, I've changed it.